### ISSUE TITLE
Remove Couchbase option

### ISSUE THREAD

**Issue by jdubois** [jhipster/generator-jhipster#8938]
I propose to remove the Couchbase option, or at least to find some "official" maintainers for it.

- Last month it was only used by 0,29% of users
- It complexifies our templates a lot, and has a high maintenance cost

So for me it's not worth having the maintenant burden for the core team, as we have tons of opened issues and requested features that many people want.

At the moment this isn't broken, so we can keep the code "as is" until either:

- It breaks and we remove it (only remove the option first, then the code 6 months later)
- We find "official" maintainers for this

I will point this tickets to a couple of people I know that might be interested, if you are interested in maintaining this part please add comments!

**Comment by DanielFran** [comment]
@PierreBesson This one is for Couchbase and https://github.com/jhipster/generator-jhipster/issues/8939 is for Cassandra.

**Comment by PierreBesson** [comment]
Sorry I didn't notice. Then I think we need to ping @tchlyah.

**Comment by tchlyah** [comment]
I've always tried to maintain this part ASAP. I'm also working on the Reactive part will I hope will be ready soon. So Yes, You can rely on me. Just ping me whenever you need me.

If you want "official" maintainer from Couchbase, I heard that @deniswsrosa is interested.

**Comment by jdubois** [comment]
Thanks @tchlyah ! And sorry as this was a bit unfair to you: at the moment we only have issues with Cassandra, and Couchbase works really well.

But yes it would be good to have an "official" maintainer, and that could be you, so we know to who we must assign the issues. -> who would like to do be the official maintainer?

**Comment by pascalgrimaud** [comment]
I don't think we want official maintainer from Couchbase directly, simply official maintainer who knows Couchbase and who wants to maintain this part in JHipster. It could be you, if you want @tchlyah

**Comment by tchlyah** [comment]
Yes sure! I would be happy to so. I think this was the condition before even you accept my PR.

**Comment by jdubois** [comment]
Yes but this time I would like to do something more official. I'll propose this to the core dev team for a vote: we should have one official leader for each important option (also for Angular, React and Vue for example)

**Comment by deniswsrosa** [comment]
Hey Folks!
 I am little late to the party, but I can also help maintain it. Let me know when you have any updates.

**Comment by cbornet** [comment]
@tchlyah 

> I've always tried to maintain this part ASAP. I'm also working on the Reactive part, I hope it will be ready soon.

May I assign the task to you in #7608 ?
